neuralmonkey/encoders/sentence_encoder.py

```diff
@@ -80,20 +80,25 @@ def __init__(self,
             .format(self.name))
 
         with tf.variable_scope(self.name):
-            self._create_input_placeholders()
-            self._create_embedding_matrix()
+            with tf.name_scope('intput_embeddings'):
```

Comment:
intput.. a fakt chcem input placeholders v input embeddings? to mi nepřijde dvakrát přehledný